### PR TITLE
Add custom assertions

### PR DESCRIPTION
**Pull request by uhafner**

### CODE REVIEW

File: src/test/java/io/jenkins/plugins/github/checks/api/ChecksDetailsTest.java [hunk 1]
```diff
@@ -22,10 +22,9 @@ void shouldCreateWhenBuildOnlyWithConstructor() {
         ChecksDetails details = new ChecksDetailsBuilder(CHECK_NAME, ChecksStatus.QUEUED)
                 .build();
 
-        assertThat(details.getName()).isEqualTo(CHECK_NAME);
-        assertThat(details.getStatus()).isEqualTo(ChecksStatus.QUEUED);
-        assertThat(details.getDetailsURL().isPresent()).isFalse();
-        assertThat(details.getConclusion().isPresent()).isFalse();
+        assertThat(details).hasName(CHECK_NAME).hasStatus(ChecksStatus.QUEUED);
+        assertThat(details.getDetailsURL()).isPresent();
+        assertThat(details.getConclusion()).isPresent();
```
Comment (XiongKezhi) [inline src/test/java/io/jenkins/plugins/github/checks/api/ChecksDetailsTest.java:27]:
```suggestion
       assertThat(details.getDetailsURL()).isNotPresent();
       assertThat(details.getConclusion()).isNotPresent();
```

**Comment by uhafner** [comment]
This was just meant as an example... so feel free to close and adapt it inline the existing PR!

**Comment by XiongKezhi** [comment]
> 
> 
> This was just meant as an example... so feel free to close and adapt it inline the existing PR!

The templates is somewhat hard for me to understand, it is much more generic than the example in "quick start", so I'll merge this to avoid mistakes while handling it :)

**Comment by uhafner** [comment]
Ah yes, I did the same 😉 
The template is copied from the AssertJ library. With 2 small modifications.

**Pull request merged**